src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraClientImpl.java

```diff
@@ -0,0 +1,184 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
+
+import java.net.URL;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.UUID;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.PaymentServiceConstants.BamboraTransType;
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.exception.PaymentServiceException;
+
+/**
+ * Bambora Client class Implementation
+ * 
+ * May be expanded in the future to include other types of Bambora transactions. 
+ * 
+ * @author smillar
```

Comment:
dumb small thing: should this reference your github account?

Reply:
I will set this as a note to myself to do. Trying to get the others devs going again.